Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java

```diff
@@ -521,8 +546,14 @@ private Bundle maybeCreateExtraInfoBundle(final BaseAd baseAd)
 
         if ( TextUtils.isEmpty( creativeId ) ) return null;
```

Comment:
Hi. Here’s another observation: if the `creativeId` is empty, the banner size won't be provided in the `extraInfo`.

Reply:
Nice catch, updated.